jcl/src/java.base/share/classes/com/ibm/jit/JITHelpers.java

```diff
@@ -69,6 +70,49 @@ private static JITHelpers jitHelpers() {
 		return helpers;
 	}
 
+	/*
+	 * Public methods to provide access to expected exception
+	 */
+
+	public static void setExpectedException(Class<? extends Throwable> expectedException) {
+		Thread currentThread = Thread.currentThread();
+		String vmThreadName = currentThread.getName();
+		initExceptionMap();
+		
+		if (expectedException == null) {
+			System.out.println("Removing exception for thread: " + vmThreadName);
```

Comment:
We do not need to print the message everytime it adds or removes the exception in the map. This would affect the timing as well.